media/static-bundles.json

```diff
@@ -570,12 +570,6 @@
       ],
       "name": "firefox_whatsnew_67"
     },
-    {
-      "files": [
-        "css/firefox/whatsnew/whatsnew-67.0.5.scss"
-      ],
-      "name": "firefox_whatsnew_67.0.5"
```

Comment:
Looks like this was accidentally deleted. If you rebase from the current master branch it should restore it, though might cause some merge conflicts you'll need to resolve manually.

Reply:
this should be fixed now